server.js

```diff
@@ -240,22 +240,28 @@ app.prepare().then(async () => {
       publicOnly: true,
     });
 
-    const exploreSlates = await Data.getSlatesByIds("id", [
-      //NOTE(tara): slates in localhost for testing
-      "857ad84d-7eff-4861-a988-65c84b62fc23",
-      "81fa0b39-0e96-4c7f-8587-38468bb67cb3",
-      "c4e8dad7-4ba0-4f25-a92a-c73ef5522d29",
-      "df05cb1f-2ecf-4872-b111-c4b8493d08f8",
-      "435035e6-dee4-4bbf-9521-64c219a527e7",
-      "ac907aa3-2fb2-46fd-8eba-ec8ceb87b5eb",
+    const exploreSlates;
```

Comment:
```
    const exploreSlates;

    if (process.env.NODE_ENV !== "production") {
      exploreSlates = await Data.getSlatesByIds("id", [
        //NOTE(tara): slates in localhost for testing
        "857ad84d-7eff-4861-a988-65c84b62fc23",
        "81fa0b39-0e96-4c7f-8587-38468bb67cb3",
        "c4e8dad7-4ba0-4f25-a92a-c73ef5522d29",
        "df05cb1f-2ecf-4872-b111-c4b8493d08f8",
        "435035e6-dee4-4bbf-9521-64c219a527e7",
        "ac907aa3-2fb2-46fd-8eba-ec8ceb87b5eb",
      ]);
    }

    const exploreSlates = await Data.getSlatesByIds("id", [
      //NOTE(tara): slates in prod
      "d2861ac4-fc41-4c07-8f21-d0bf06be364c",
      "9c2c458c-d92a-4e81-a4b6-bf6ab4607470",
      "7f461144-0647-43d7-8294-788b37ae5979",
      "f72c2594-b8ac-41f6-91e0-b2da6788ae23",
      "a0d6e2f2-564d-47ed-bf56-13c42634703d",
      "0ba92c73-92e7-4b00-900e-afae4856c9ea",
    ]);

```

This snippet of code won't work for a number of reasons:

* `const exploreStates` can not be re-assigned as a variable. The only reason we use `const` is to signal that this variable will not change later in the function.
* For other people's local environments, your slates will not exist, does the function still return an empty array after it is called?
* What you're looking for is to use `let` and to avoid the reassignment.

Reply:
I recommend you do something like

```
let exploreSlates = [];

if (Environment.IS_PRODUCTION) {
  exploreSlates = await Data.getSlatesByIds("id", [
      //NOTE(tara): slates in prod
      "d2861ac4-fc41-4c07-8f21-d0bf06be364c",
      "9c2c458c-d92a-4e81-a4b6-bf6ab4607470",
      "7f461144-0647-43d7-8294-788b37ae5979",
      "f72c2594-b8ac-41f6-91e0-b2da6788ae23",
      "a0d6e2f2-564d-47ed-bf56-13c42634703d",
      "0ba92c73-92e7-4b00-900e-afae4856c9ea",
    ]);
} else {
      exploreSlates = await Data.getSlatesByIds("id", [
        //NOTE(tara): slates in localhost for testing
        "857ad84d-7eff-4861-a988-65c84b62fc23",
        "81fa0b39-0e96-4c7f-8587-38468bb67cb3",
        "c4e8dad7-4ba0-4f25-a92a-c73ef5522d29",
        "df05cb1f-2ecf-4872-b111-c4b8493d08f8",
        "435035e6-dee4-4bbf-9521-64c219a527e7",
        "ac907aa3-2fb2-46fd-8eba-ec8ceb87b5eb",
      ]);
}
```

Also `Data.getSlatesByIds` should not take two arguments. It should just take an array such as:

`Data.getSlatesByIds([1, 2, 3, 4, 5])`;

Please modify the function to behave this way.

Reply:
I see ok

Reply:
just pushed, should work this time. thanks!